README.md

```diff
@@ -1,24 +1,133 @@
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Ruby version
+# 📖 [Blog-app] <a name="about-project"></a>
 
-* System dependencies
 
-* Configuration
+**[Blog-app]** this is a blog application where users can upload a post, comment and like the post.
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
 
-* How to run the test suite
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+<summary>Back-end</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **Create user**
+- **Create post**
+- **Like post**
+- **Comment post**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+ setup PostgreSql
+ Visual Studio Code
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/shaaibu7/Blog-app
+  cd Blog-app
+```
+
+### Usage
+
+This repository includes files with database migrations that can be used to recreate a database:
+
+- Use [schema.sql](./db/schema.rb) to create all tables.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Shaaibu Suleiman**
+
+- GitHub: [@githubhandle](https://github.com/shaaibu7)
+- Twitter: [@twitterhandle](https://twitter.com/SuleimanShaaibu?t=EAqVJrP59poEb2W46kK9vg&s=09)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shaaibu-suleiman-119271206)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Implement dashboard to manage blog**
+- [ ] **Improved user interface**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project then don't forget to give star ⭐ to this repository.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for the learning materials and technical support provided to work on this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] @shaaibu7 great work on your documentation, however, your MIT License link is not working, this is because you are referencing `./MIT` instead of `/LICENSE ` in your link, kindly consider changing it.